build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.4.0-alpha7'
```

Comment:
i don't normally depend on the alpha versions of the android build tools and i'm not sure if i want to do it for this demo project. is this required for one of the changes you wanted to merge in?